src/markdown-pages/docs/managers/core-manager.md

```diff
@@ -3,12 +3,144 @@ path: "/docs/managers/core-manager"
+compileOnly 'dev.galasa:dev.galasa.core.manager'
+}
+```
+
+
+Finally, use the annotation that is provided by the Manager in the test. The annotation calls the Manager at runtime. In this example, the provided annotation is _@CoreManager_. 
+
+
+# <a name="configuring"></a>Configuring 
+
+The following are properties used to configure the Core Manager:
+
+## <a name="cps"></a>Configuration Properties
+
+<details>
+<summary>Resource String Pattern CPS Property</summary>
+
+| Property: | Resource String Pattern CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | core.resource.string.[length].pattern |
+| Description: | Sets the patterns of resources strings for a length of string.  The patterns are from the Galasa ResourcePoolingService which uses a homegrown syntax.  |
+| Required:  | No |
+| Default value: | {A-Z} for each byte for the specified length |
+| Valid values: | For each character can be a constant or a random choice from a literal, eg {A-Z results in a single character between A and Z inclusive. {0-9} or {a-zA-Z0-9} are options. DFH{A-Z}{0-1}{0-9}{0-9}{0-9}, results in DFHA1789 for example, the 4th character can only be 0 or 1. 
+| Examples: | <code>core.resource.string.8.length={A-Z}{A-Z}{A-Z}{A-Z}{A-Z}{A-Z}{A-Z}{A-Z}<br> </code> |
+
+</details>
+
+# <a name="annotations"></a>Annotations provided by the Manager
+
+The following annotations are provided by the Core Manager:
+
+<details>
+<summary>Logger</summary>
+
+| Name: | Logger |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @Logger |
+| Description: | Creates the log which is then automatically stored in the Result Archive Store (RAS) by the Galasa framework. |
+| Syntax:  | logger.info ("message");|
+
+</details>
+
+<details>
+<summary>Stored Artifact Root</summary>
+
+| Name: | StoredArtifactRoot |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @StoredArtifactRoot |
+| Description: | Lets your test write specific output to the RAS. |
+| Syntax:  | artifactRoot.resolve(folder).resolve(file);|
+
+</details>
+
+<details>
+<summary>Test Property</summary>
+
+| Name: | TestProperty |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @TestProperty |
+| Description: |  |
+| Syntax:  | |
+
+</details>
+
+
+
+# <a name="codesnippets"></a>Code snippets and examples
+
+<details><summary>Extract credentials from a file</summary>
+
+You can extract credentials by using the `getCredentials` method. The Core Manager uses the `getCredentials` method to retrieve a user id and password from a file to use in your test.
+
+```
+import dev.galasa.ICredentials;
+import dev.galasa.ICredentialsUsernamePassword;
+...
+@Test
+...
+ICredentialsUsernamePassword credentials = (ICredentialsUsernamePassword) \
+coreManger.getCredentials("APP");
+credentials.getPassword();
+credentials.getUsername();
```

Comment:
We should show how the user should update the credentials store to enter the username and password that will be picked up by this call

Reply:
updated with SIMBANK